index.html

```diff
@@ -1,579 +1,998 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title>
-DID Implementation Guide v1.0
-  </title>
-  <meta content='text/html; charset=utf-8' http-equiv='Content-Type'><!--
-    === NOTA BENE ===
-    For the three scripts below, if your spec resides on dev.w3 you can check them
-    out in the same tree and use relative links so that they'll work offline.
-   -->
-
-  <script class='remove'
-    src='https://www.w3.org/Tools/respec/respec-w3c-common'>
-  </script>
-  <script src='./common.js' class='remove'></script>
-  <script class="remove" type="text/javascript">
-var respecConfig = {
-      wgPublicList: "public-did-wg",
-      wgPatentURI:  "https://www.w3.org/2004/01/pp-impl/117488/status",
-      wg: "Decentralized Identifier Working Group",
-      wgURI: "https://www.w3.org/2019/did-wg/",
-
-  // specification status (e.g., WD, LCWD, NOTE, etc.). If in doubt use ED.
-  specStatus: "ED",
-
-  // the specification's short name, as in http://www.w3.org/TR/short-name/
-  shortName: "did-imp-guide",
-
-  edDraftURI: "https://w3c.github.io/did-imp-guide/",
-
-  // subtitle
-  subtitle: "An implementation guide for DID software developers",
-
-  // if you wish the publication date to be other than today, set this
-  //publishDate:  "2019-11-07",
-
-  // if there is a previously published draft, uncomment this and set its YYYY-MM-DD date
-  // and its maturity status
-  // previousPublishDate:  "1977-03-15",
-  // previousMaturity:  "WD",
-
-  // extend the bibliography entries
-  localBiblio: {
-    "LD-PROOFS": {
-      title: "Linked Data Proofs",
-      href: "https://w3c-dvcg.github.io/ld-proofs/",
-      authors: [
-        "Manu Sporny",
-        "Dave Longley"
+  <title>DID Implementation Guide v1.0</title>
+  <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
+  <!--
+      === NOTA BENE ===
+      For the three scripts below, if your spec resides on dev.w3 you can check them
+      out in the same tree and use relative links so that they'll work offline,
+     -->
+  <script class="remove" src="https://www.w3.org/Tools/respec/respec-w3c"></script>
+  <script class="remove" src="https://unpkg.com/reqlist/lib/reqlist.js"></script>
```

Comment:
This link looks fragile, and is also unused.  As per the instructions, the functions are not hooked up and css is not loaded.

Reply:
Opened https://github.com/w3c/did-imp-guide/issues/4